system/lib/pthread/emscripten_futex_wait.c

```diff
@@ -116,7 +116,7 @@ int emscripten_futex_wait(volatile void *addr, uint32_t val, double max_wait_ms)
     return -EINVAL;
   }
 
-  _emscripten_yield();
+  _emscripten_yield(0);
```

Comment:
```suggestion
  // We don't know the current time, so pass 0.
  _emscripten_yield(0);
```

Reply:
This might make reading the code easier, as it's not obvious without reading the header. But up to you.